### PR TITLE
Docker integration tests: factorize setup

### PR DESCRIPTION
**Pull request by pilou-**
##### SUMMARY
docker_container: factorize setup for docker integration tests

- ~docker_container: add integration tests~ (will be done in another PR)
- factorize setup for docker integration tests
- docker_swarm: enable RHEL builds

##### ISSUE TYPE
 - Bugfix Pull Request

##### COMPONENT NAME
docker integration test

##### ANSIBLE VERSION
```
2.7
```

##### ADDITIONAL INFORMATION

```

```

### CODE REVIEW

**Comment by tbouvet** [comment]
@pilou-  Good idea to factorize a setup for docker tests.

File: test/integration/targets/docker_swarm/aliases [hunk 1]
```diff
@@ -1,5 +1,4 @@
 posix/ci/group2
```
Comment (felixfontein) [inline test/integration/targets/docker_swarm/aliases]: The group names have changed recently (yesterday or so?). I think it should be `shippable/cloud/group2`; is that correct, @mattclay?

Comment (felixfontein) [reply test/integration/targets/docker_swarm/aliases]: (Though I guess this will be fixed automatically when merging/rebasing to current devel.)

**Review by felixfontein — APPROVED** [review]
LGTM
There should be tests for the other docker modules as well (in particular docker_container), and this lays the groundwork for that.

**Review by tbouvet — APPROVED** [review]
LGTM
OK for docker_swarm.

**Comment by pilou-** [comment]
@felixfontein @tbouvet PR updated:
- rebased on devel
- debug commit removed
Another PR will bring tests for docker_container module.

Could you approve again this PR (ansibot ignores previous approvals due to the rebase) ?

**Comment by felixfontein** [comment]
@pilou- Are you sure you rebased against current `devel`? On `devel`, [test/integration/targets/docker_swarm/aliases](https://github.com/ansible/ansible/blob/devel/test/integration/targets/docker_swarm/aliases) starts with `shippable/posix/group2`, while in [your tree](https://github.com/pilou-/ansible/blob/docker_integration_tests/test/integration/targets/docker_swarm/aliases), it starts with `posix/ci/group2`.

**Comment by pilou-** [comment]
oups, I rebased again: it seems ok now. Thanks for catching that!

**Review by felixfontein — APPROVED** [review]
shipit

**Comment by pilou-** [comment]
@tbouvet could you comment this PR with `shipit` again ? Your previous approval is ignored due to the rebase.

**Review by mattclay — APPROVED** [review]

**Pull request merged**